### ISSUE TITLE
[Bug]: Single node view section positioning flickering in between container sections

### ISSUE THREAD

**Issue by hugovalente-pm** [netdata/netdata-cloud#438]
### Bug description

While navigating through (k8s) container sections the FE seems to get in a state it can't determine to which section to jump to.

### Expected behavior

Position the user in one of the sections, the flickering shouldn't happen

### Steps to reproduce

1. Go to a Single node view on a k8s container section- [example](https://app.netdata.cloud/spaces/netdata-cloud/rooms/k8s-prd/nodes/65587106-12b0-4e7b-ad6b-17fb3c1915df#chartName=menu_cgroup_k8s_cntr_app_cloud-agent-service-0-588f77f5d9-lzkht_cloud-agent-service&appliedFilteredIds=&reachableNodesIds=&staleNodesIds=&unreachableNodesIds=&filteredNodesSearchParams=&metrics_correlation=false&after=-900&before=0&utc=Europe%2FBucharest&offset=%2B3&timezoneName=E.%20Europe)
2. Do a single small scroll up and see that the section starts flickering

### Screenshots

![chrome_M6HtJcqpid](https://user-images.githubusercontent.com/82235632/170125907-4ef8397c-2db4-4897-854c-6017ab6ed408.gif)

### Desktop

OS: Windows 10
Browser: Chrome
Browser Version: 101.0.4951.67 (Official Build) (64-bit)

**Comment by dimko** [comment]
I've also noticed this in my tests for non k8s charts. Unfortunately we raised the issue in a not so proper way.
Video attached in the ticket below showcasing the behavior https://github.com/netdata/cloud-frontend/issues/3395

**Comment by hugovalente-pm** [comment]
thanks @dimko for sharing that, it doesn't seem to be just for k8s charts, let me rephrase this

**Comment by cakrit** [comment]
Actually I was wrong, users have a workaround for this one, it's just annoying, not blocking work.

**Comment by novykh** [comment]
@dimko is this still reproducible?

**Comment by dimko** [comment]
Yes, just did it and almost suffered from epilepsy. 

![image](https://user-images.githubusercontent.com/19268853/186427567-229525d0-d771-4d2b-a133-fea21a926848.png)

**Comment by hugovalente-pm** [comment]
closing this since it seems to have been fixed with the refactoring
